core/posts.go

```diff
@@ -478,12 +506,14 @@ func validatePost(post *pb.Post) (err error) {
 		return errors.New("Slugs cannot contain file separators")
 	}
 
-	// Tile
-	if post.Title == "" {
-		return errors.New("Post must have a title")
+	// Type
+	if _, ok := pb.Post_PostType_value[post.PostType.String()]; !ok {
+		return errors.New("Invalid post type")
```

Comment:
We're trying to export these errors so they can be checked against outside of the package. We prefix our error constants with `Err<DescriptiveNameOfError>`. So this might be `var ErrInvalidPostType = errors.New("...")`.

Reply:
Done.